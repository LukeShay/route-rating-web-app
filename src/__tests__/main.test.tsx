import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Main from '../main';

describe('<Main />', () => {
  it('should render correctly', () => {
    const output = shallow(<Main />);
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
