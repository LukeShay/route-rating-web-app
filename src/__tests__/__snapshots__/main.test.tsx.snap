// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render correctly 1`] = `
<Memo(StoreCombiner)
  stores={
    Array [
      [Function],
      [Function],
      [Function],
    ]
  }
>
  <BrowserRouter>
    <App />
  </BrowserRouter>
</Memo(StoreCombiner)>
`;
